src/index.test.ts

```diff
@@ -91,6 +91,7 @@ describe('index', () => {
         "definePattern",
         "exactOptional",
         "getChecksumAddress",
+        "getChecksumAddressUnmemoized",
```

Comment:
Not sure if this should be exported.

Reply:
Not sure either, I am wondering if anyone might want to use unmemoized version

Reply:
export removed